src/System.Reflection.MetadataLoadContext/src/Resources/Strings.resx

```diff
@@ -0,0 +1,273 @@
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="Argument_GenericArgsCount" xml:space="preserve">
+    <value>The number of generic arguments provided does not match the number of generic parameters.</value>
+  </data>
+  <data name="Argument_HasToBeArrayClass" xml:space="preserve">
+    <value>Must be an array type.</value>
+  </data>
+  <data name="Argument_InvalidEnum" xml:space="preserve">
+    <value>The Enum type should contain one and only one instance field.</value>
+  </data>
+  <data name="Arg_AmbiguousMatchException" xml:space="preserve">
+    <value>Ambiguous match found.</value>
+  </data>
+  <data name="Arg_EmptyArray" xml:space="preserve">
+    <value>Array may not be empty.</value>
+  </data>
+  <data name="Arg_EnumLitValueNotFound" xml:space="preserve">
+    <value>Literal value was not found.</value>
+  </data>
+  <data name="Arg_HTCapacityOverflow" xml:space="preserve">
+    <value>Hashtable's capacity overflowed and went negative. Check load factor, capacity and the current size of the table.</value>
+  </data>
+  <data name="Arg_MustBeEnum" xml:space="preserve">
+    <value>Type provided must be an Enum.</value>
+  </data>
+  <data name="Arg_MustBeType" xml:space="preserve">
+    <value>Type must be a type provided by the MetadataLoadContext.</value>
+  </data>
+  <data name="Arg_NotGenericMethodDefinition" xml:space="preserve">
+    <value>{0} is not a GenericMethodDefinition. MakeGenericMethod may only be called on a method for which MethodBase.IsGenericMethodDefinition is true.</value>
+  </data>
+  <data name="Arg_NotGenericParameter" xml:space="preserve">
+    <value>Method may only be called on a Type for which Type.IsGenericParameter is true.</value>
+  </data>
+  <data name="Arg_NotGenericTypeDefinition" xml:space="preserve">
+    <value>{0} is not a GenericTypeDefinition. MakeGenericType may only be called on a type for which Type.IsGenericTypeDefinition is true.</value>
+  </data>
+  <data name="Arg_ReflectionOnlyCA" xml:space="preserve">
+    <value>The requested operation is invalid on objects loaded by a MetadataLoadContext. Use CustomAttributeData instead.</value>
+  </data>
+  <data name="Arg_ReflectionOnlyIllegal" xml:space="preserve">
+    <value>The requested operation is invalid on objects loaded by a MetadataLoadContext.</value>
+  </data>
+  <data name="Arg_ReflectionOnlyInvoke" xml:space="preserve">
+    <value>It is illegal to invoke a method on a Type loaded by a MetadataLoadContext.</value>
```

Comment:
Nit, we generally avoid the term `illegal`. Something like "Cannot invoke a method .." is quicker to read anyway.

Same for the other uses

Reply:
Done